charts/incubator/nitter/templates/_configmap.tpl

```diff
@@ -0,0 +1,52 @@
+{{/* Define the configmap */}}
+{{- define "nitter.configmap" -}}
+
+{{- $configName := printf "%s-nitter-config" (include "tc.common.names.fullname" .) }}
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ $configName }}
+  labels:
+    {{- include "tc.common.labels" . | nindent 4 }}
+data:
+  nitter.conf: |
+    [Server]
+    address = "0.0.0.0"
+    port = {{ .Values.service.main.ports.main.port }}
```

Comment:
@stavros-k i done it here.

Reply:
I see. 
hostname confused me, as 0.0.0.0 is not valid. It's only used when you want to bind to all interfaces

Reply:
address and port leave it as is